views/components/Certification.js

```diff
@@ -1,72 +1,388 @@
+  const handleClick3 = () => setSelectedItem(INIT_CERT_ACTIVE_ITEM[2])
+  const handleClick4 = () => setSelectedItem(INIT_CERT_ACTIVE_ITEM[3])
+
+  const Research = () => {
+    return (
+      <div>
+        <ResearchItem onClick={handleClick1}>
+          <ItemIcon1 itemSelected={itemSelected} />
+        </ResearchItem>
+
+        <ResearchInfo itemSelected={itemSelected}>
+          <Header>
+            <span>1</span> &mdash; Bachelor of Science
+          </Header>
+          <p>
+            Graduating from HCM University of Science, Advanced Program in
+            Computer Science.
+          </p>
+        </ResearchInfo>
+      </div>
+    )
+  }
+
+  const Develop = () => {
+    return (
+      <div>
+        <DevelopItem onClick={handleClick2}>
+          <ItemIcon2 itemSelected={itemSelected} />
+        </DevelopItem>
+
+        <DevelopInfo itemSelected={itemSelected}>
+          <Header>
+            <span>2</span> &mdash; Communicative Certification: TOEFL ibt 85 pts
+          </Header>
+          <p>
+            The TOEFL iBT® test measures your ability to use and understand
+            English at the university level. And it evaluates how well you
+            combine your reading, listening, speaking and writing skills to
+            perform academic tasks.
+          </p>
+        </DevelopInfo>
+      </div>
+    )
+  }
+
+  const Deploy = () => {
+    return (
+      <div>
+        <DeployItem onClick={handleClick3}>
+          <ItemIcon3 itemSelected={itemSelected} />
+        </DeployItem>
+
+        <DeployInfo itemSelected={itemSelected}>
+          <Header>
+            <span>3</span> &mdash; Second place
+          </Header>
+          <p>
+            Got second prize for the competitive algorithm contest of the
+            university
+          </p>
+        </DeployInfo>
+      </div>
+    )
+  }
+
+  const Deliver = () => {
+    return (
+      <div>
+        <DeliverItem onClick={handleClick4}>
+          <ItemIcon4 itemSelected={itemSelected} />
+        </DeliverItem>
+
+        <DeliverInfo itemSelected={itemSelected}>
+          <Header>
+            <span>4</span> &mdash; First place
+          </Header>
+          <p>
+            Highschool, got First prize for the competitive algorithm contest of
+            the city.
+          </p>
+        </DeliverInfo>
+      </div>
+    )
+  }
+
+  return (
+    <Container id={CERTIFICATIONS}>
+      <div className="container">
+        <header className="common-section__header">
+          <h2>Certifications and Prizes</h2>
+          <h4></h4>
+        </header>
+        <div className="grid-container">
+          <List className="certification">
+            <Research />
+            <Develop />
+            <Deploy />
+            <Deliver />
+          </List>
+        </div>
       </div>
-    </div>
-  </section>
-)
+    </Container>
+  )
+}
+
+const ItemIcon1 = ({ itemSelected }) => {
```

Comment:
using the passing  `id` to compare instead of fixed  INIT_CERT_ACTIVE_ITEM

Reply:
Updated